src/components/views/messages/TextualBody.js

```diff
@@ -80,6 +93,15 @@ module.exports = React.createClass({
                     }
                 }, 10);
             }
+            const buttons = ReactDOM.findDOMNode(this).getElementsByClassName("mx_EventTile_copyButton");
+            if (buttons.length > 0) {
```

Comment:
Also this check is now redundant, since if the length is 0, the for loop will just run zero times.